samples-azure-functions/src/test/java/com/functions/EndToEndTests.java

```diff
@@ -14,6 +14,8 @@
 public class EndToEndTests {
     private static final String hostHealthPingPath = "/admin/host/ping";
     private static final String startOrchestrationPath = "/api/StartOrchestration";
+    private static final String approvalWorkFlow = "/api/ApprovalWorkflowOrchestration";

```

Comment:
nit: "workflow" is one word. Also, it's good practice to be more clear about what a variable represents. In this case it doesn't represent a "workflow" but a URL for starting a workflow, so a better variable name would be `startApprovalWorkflowUrl`
```suggestion
    private static final String startApprovalWorkflowUrl = "/api/ApprovalWorkflowOrchestration";

```